atc/radar/resource_scanner.go

```diff
@@ -155,8 +166,14 @@ func (scanner *resourceScanner) scan(logger lager.Logger, resourceID int, fromVe
 		return 0, err
 	}
 
+	// Combine pipeline specific var_sources with the global credential manager.
+	varss, err := scanner.dbPipeline.Variables(logger, scanner.variables, scanner.varSourcePool)
```

Comment:
Ah, I see, this part only has `scanner.variables`, not a secrets manager. Maybe that could be changed. We're going to phase out `radar/` soon-ish anyway. :thinking:

Reply:
Changed radar to pass secrets instead of variables.